public/keymaps/treadstone32_default.json

```
@@ -0,0 +1,33 @@
+{
+    "keyboard": "treadstone32",
+    "keymap": "treadstone32_default",
+    "layout": "LAYOUT",
+    "layers": [
+        [
+            "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_Y", "KC_U", "KC_I", "KC_O", "KC_P",
+            "LSFT_T(KC_A)", "KC_S", "KC_D", "KC_F", "KC_G", "KC_H", "KC_J", "KC_K", "KC_L", "RSFT_T(KC_ENT)",
+            "LCTL_T(KC_Z)", "LALT_T(KC_X)", "LGUI_T(KC_C)", "KC_V", "KC_B", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "RCTL_T(KC_SLSH)",
+            "LT(1,KC_BSPC)", "LT(2,KC_SPC)"
+        ],
+        [
+            "KC_F1", "KC_F2", "KC_F3", "KC_F4", "KC_F5", "KC_MINS", "KC_EQL", "KC_LBRC", "KC_RBRC", "KC_BSLS",
+            "LSFT_T(KC_F6)", "KC_F7", "KC_F8", "KC_F9", "KC_F10", "KC_NO", "KC_NO", "KC_NO", "KC_SCLN", "KC_QUOT",
+            "LSFT_T(KC_F11)", "LALT_T(KC_F12)", "KC_ESC", "KC_TAB", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_NO", "KC_GRV",
```

The second `KC_NO` entry on this line should be a `KC_DEL`:

```suggestion
            "LSFT_T(KC_F11)", "LALT_T(KC_F12)", "KC_ESC", "KC_TAB", "KC_NO", "KC_DEL", "KC_NO", "KC_NO", "KC_NO", "KC_GRV",
```